src/components/courses/components/course-card/courses-cards.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button, List} from "reactstrap";
+import toHoursAndMinutes from "../../../../common/to-hours-and-minutes";
+
+function CoursesCards({courses, authors}) {
+
+	function authorsNameById(authorsId) {
+		return authorsId.map(authorId => authors.find(({ id }) => id === authorId)?.name).join(', ')
+	}
+
+	return (
+		<List className='mt-1'>
+			{courses.map((course) => {
+				return (
+					<li key={course.id} className='row border border-success mt-1 mx-0'>
+						<dl className='col-7 m-1' >
+							<dt><b>{course.title}</b></dt>
```

Comment:
It should not be `dl` in this case, just `h2/h3` and `p`

Reply:
Done!